config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -388,6 +412,7 @@ presubmits:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-upstream-k8s-ci-windows-containerd-serial-slow-main
   - name: pull-cluster-api-provider-azure-apidiff
+    cluster: eks-prow-build-cluster
     decorate: true
     decoration_config:
       gcs_credentials_secret: "" # Use workload identity for uploading artifacts
```

Comment:
@CecileRobertMichon @ameukam are there any concerns with `decoration_config.gcs_credentials_secret`? Will that be alright to move to eks?

Reply:
Looking at this I think it was mostly likely copy paste from the [cluster-api jobs](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml#L25), I don't think it's actually used but not entirely sure...  @killianmuldoon do you know ?

Reply:
It's ok to drop `decoration_config`. It's mostly used to override the default configuration.